Runtime/BlockadeLabsSkyboxMesh.cs

```diff
@@ -181,7 +181,7 @@ private Mesh CreateMesh(int subdivisions)
         {
 #if UNITY_EDITOR
             // Look for a mesh in the project
-            var folder = AssetUtils.GetOrCreateFolder("Meshes");
+            AssetUtils.TryCreateFolder("Meshes", out var folder);
```

Comment:
GetOrCreateFolder made more sense here

Reply:
thinking more on it, I don't like the name "TryCreateFolder. It's not clear that it "fails" if the folder exists